src/services/videoconference/index.js

```diff
@@ -43,319 +43,314 @@ const { ObjectId } = require('../../helper/compare');
+	if (!userIsAllowedTo(permission, permissions)) {
+		throw new Forbidden(`permission ${permission} not given in scope`);
 	}
+}
 
-	static throweOnNotAnyPermissionInScope(requiredPermissions, scopePermissions) {
-		if (scopePermissions.filter((permission) => requiredPermissions.includes(permission)).length === 0) {
-			throw new Forbidden(`one permission of ${JSON.stringify(requiredPermissions)} required in scope`);
-		}
+function throweOnNotAnyPermissionInScope(requiredPermissions, scopePermissions) {
+	if (scopePermissions.filter((permission) => requiredPermissions.includes(permission)).length === 0) {
+		throw new Forbidden(`one permission of ${JSON.stringify(requiredPermissions)} required in scope`);
 	}
+}
 
-	/**
-	 * Takes valid and enabled options and returns the enabled toggles
-	 * @param {*} options
-	 */
-	static getValidOptions(options = {}) {
-		const validOptions = {};
-		const toggleOptions = Object.values(CREATE_OPTION_TOGGLES);
-		toggleOptions.forEach((option) => {
-			copyPropertyNameIfIncludedInValuesFromSourceToTarget({
-				source: options,
-				propertyName: option,
-				values: [true, false],
-				target: validOptions,
-				sourcePropertyNames: toggleOptions,
-			});
+/**
+ * Takes valid and enabled options and returns the enabled toggles
+ * @param {*} options
+ */
+function getValidOptions(options = {}) {
+	const validOptions = {};
+	const toggleOptions = Object.values(CREATE_OPTION_TOGGLES);
+	toggleOptions.forEach((option) => {
+		copyPropertyNameIfIncludedInValuesFromSourceToTarget({
+			source: options,
+			propertyName: option,
+			values: [true, false],
+			target: validOptions,
+			sourcePropertyNames: toggleOptions,
 		});
-		return validOptions;
-	}
+	});
+	return validOptions;
+}
 
 
-	/**
-	 * checks if the school feature is enabled
-	 * @param {String} schoolId
-	 * @returns Boolean
-	 */
-	static async isSchoolFeatureEnabled(schoolId) {
-		const school = await Schools.findById(schoolId).lean().exec();
-		if (school && school.features
-			&& Array.isArray(school.features)
-			&& school.features.includes(SCHOOL_FEATURES.VIDEOCONFERENCE)) {
-			return true;
-		}
-		return false;
+/**
+ * checks if the school feature is enabled
+ * @param {String} schoolId
+ * @returns Boolean
+ */
+async function isSchoolFeatureEnabled(schoolId) {
+	const school = await Schools.findById(schoolId).lean().exec();
+	if (school && school.features
+		&& Array.isArray(school.features)
+		&& school.features.includes(SCHOOL_FEATURES.VIDEOCONFERENCE)) {
+		return true;
 	}
+	return false;
+}
 
-	static userHasVideoconferencePermissionsInScope(userPermissions) {
-		const videoConferencePermissionValues = Object.values(PERMISSIONS);
-		const videoConferencePermissionsOfUser = userPermissions
-			.filter((permission) => videoConferencePermissionValues.includes(permission));
-		return videoConferencePermissionsOfUser.length > 0;
-	}
+function userHasVideoconferencePermissionsInScope(userPermissions) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['userHasVideoconferencePermissionsInScope' is defined but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5401970)